xbmc/playlists/PlayListXSPF.cpp

```diff
@@ -0,0 +1,127 @@
+/*
+ *  Copyright (C) 2018 Tyler Szabo
+ *  Copyright (C) 2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "PlayListXSPF.h"
+
+#include "utils/log.h"
+#include "utils/URIUtils.h"
+#include "utils/XBMCTinyXML.h"
+#include "URL.h"
+
+using namespace PLAYLIST;
+
+namespace
+{
+
+static constexpr char const* LOCATION_TAGNAME = "location";
```

Comment:
Remove `static` - you already put it in an anonymous namespace, which has the same effect